File: JavaScriptPrograms/Maximum.js
var readln = (typeof readline === 'function') ? (readline) : (function() {
   importPackage(java.io);
   importPackage(java.lang);
   var stdin = new BufferedReader(new InputStreamReader(System['in']));
   return function() {
     return String(stdin.readLine());   
   };                                    
 }());
importPackage(java.io);
importPackage(java.lang);
var readln = (typeof readline === 'function') ? (readline) : (function() {
var stdin = new BufferedReader(new InputStreamReader(System['in']));
return function() {
    return String(stdin.readLine());   
  };                                    
}());
print("To find maximum number between n numbers, enter the number: ")
var Limit = parseInt(readln())
var Numbers = [Limit]
print("Enter the first number:")
Numbers[0] = parseInt(readln())
for(var Counter = 1; Counter < Limit; Counter++) {
  print("Enter the next number:")
  Numbers[Counter] = parseInt(readln())
}
var Read = "The maximum number among "
for(var Counter = 0; Counter < Limit; Counter++) {
  Read = Read + " " + Numbers[Counter]
}
var LargerNumber = 0
var Counter = 0

while(Counter < Limit) {
  if(Numbers[Counter] > LargerNumber)
    LargerNumber = Numbers[Counter]
  Counter++
}
Read = Read + " is " + LargerNumber + "."
print(Read)

File: PythonPrograms/Maximum.py
print('To print maximum number among 10 numbers.')
List = []
FirstNumber = int(input('Enter the first numbers:'))
List.append(FirstNumber)
for Count in range(1, 10):
  NextNumber = int(input('Enter the next number:'))
  List.append(NextNumber)
print('The maximum number among the 10 numbers that you have entered is', end = '')
LargerNumber = List[0]
for Count in range(1, 10):
  if List[Count] > LargerNumber:
    LargerNumber = List[Count]
print('', LargerNumber, end = '')
print('.')